tv_apps/smart-system/js/app_install_manager.js

```diff
@@ -81,6 +99,12 @@ var AppInstallManager = {
     window.addEventListener('home', this.hideAllDialogs.bind(this));
   },
 
+  isMarketplaceAppActive: function ai_isMarketplaceAppActive() {
+    var activeApp = AppWindowManager.getActiveApp();
+    return activeApp &&
+      (activeApp.manifestURL.indexOf(this.marketplaceURLPattern) !== -1);
```

Comment:
I would suggest hard code marketplace app uri in case partner's marketplace also has this keyword.